packages/pwa-buildpack/envVarDefinitions.json

```
@@ -31,6 +31,12 @@
                     "type": "str",
                     "desc": "Specify the Magento store view code set in Stores > Settings > All Stores > Store View name",
                     "default": ""
+                },
+                {
+                    "name": "USE_STORE_CODE_IN_URL",
+                    "type": "bool",
+                    "desc": "Appends the store code to the store URLs if value is true.",
```

```suggestion
                    "desc": "Includes the store code in the store URLs if value is true.",
```

I suggest that because we are technically not appending to the URLs, we are appending to the store's domain.